documentation/docs/user-guide/user-officer/user-officer.md

```diff
@@ -0,0 +1,114 @@
+# User Officer
+
+User officer
+
+Welcome to the User Office documentation for **User Officers!**
+
+
+![workflow](images/call.png){ width="300" } ![workflow](images/workflow.png){ width="300" }
+
+
+List of all how-to guides: 
+
+### **Calls**
+* #### [How do I create a call?](2create_call.md) 
+
+### **Managing proposals**
+
+* #### [How do I edit a users' proposal?](4edit_propoal.md)
+
+* #### [How do I impersonate users?](impersonate.md)
+
+* #### [How do I revert submitted proposals back to draft status?](4edit_proposal.md)
+
+### **Creating templates**
+
+* #### [How do I create a proposal template?](2proposal_template.md)
+
+* #### [How do I create a feedback template?](feedback_template.md)
+
+* #### [How do I create a PDF template?](pdf_template.md)
+
+* #### [How do I create a proposal ESI template?](proposalESI_template.md)
+
+* #### [How do I create a question template?](question_template.md)
+
+* #### [How do I create a sample declaration template?](sampledec_template.md)
+
+* #### [How do I create a sample ESI template?](sampleESI_template.md)
+
+* #### [How do I create a shipment declaration template?](shipment_template.md)
+
+* #### [How do I create a sub template?](sub_template.md)
+
+* #### [How do I create a visit registration template?](visit_template.md)
+
+* #### [How do I create a workflow template?](workflow_template.md)
+
+### **Feasibility review**
+
+* #### [How do I configure the system for feasibility review?](feasibility_review.md)
+
+### **FAP**
+* #### [How do I create a FAP?](fap.md)
+* #### [How do I configure the system for FAP review?](fap.md)
+ 
+### **Editing templates**
+
+* #### [How do I edit a proposal template?](2proposal_template.md)
+
+* #### [How do I edit a FAP?](fap_template.md)
+
+* #### [How do I edit a feedback template?](feedback_template.md)
+
+* #### [How do I edit a PDF template?](pdf_template.md)
+
+* #### [How do I edit a proposal ESI template?](proposalESI_template.md)
+
+* #### [How do I edit a question template?](question_template.md)
+
+* #### [How do I edit a sample declaration template?](sampledec_template.md)
+
+* #### [How do I edit a sample ESI template?](sampleESI_template.md)
+
+* #### [How do I edit a shipment declaration template?](shipment_template.md)
+
+* #### [How do I edit a sub template?](sub_template.md)
+
+* #### [How do I edit a visit registration template?](visit_template.md)
+
+* #### [How do I edit a workflow template?](workflow_template.md)
+
+
+## **Configure the system**
+
+
+* #### [How do I change the look of the UI?](UI.md)
+
+
+
+## **Dev tools**
+
+* #### [How do I use API access tokens?](API.md)
+
+* #### [How do I use features?](features.md)
+* #### [How do I use App settings?](app_settings)
+* #### [How do I edit a PDF](API.md)
+ 
+
+# Glossary
+
+* [What are templates?](templates_info)
+* [What are workflows?](workflows_info)
+* [What are proposal templates?](2proposal_template.md)
```

Comment:
Some links are not updated, I guess: in this case, `templates/proposal_template.md`.